test-container/src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -94,13 +118,18 @@ protected void doStart() {
         super.doStart();
     }
 
+    public StrimziKafkaContainer withExternalZookeeper(String connectString) {
```

Comment:
When I made the comment here before about the factory methods I was expecting the external ZK to be one of them. Now I see that you've got three overloads for those methods. Another 3 seems a bit silly, but maybe the external ZK should just be extra parameter for a 4th factory?